pkg/skaffold/update/update.go

```diff
@@ -44,15 +45,15 @@ func IsUpdateCheckEnabled() bool {
 
 // GetLatestAndCurrentVersion uses a VERSION file stored on GCS to determine the latest released version
 // and returns it with the current version of Skaffold
-func GetLatestAndCurrentVersion() (semver.Version, semver.Version, error) {
+func GetLatestAndCurrentVersion(url string) (semver.Version, semver.Version, error) {
 	none := semver.Version{}
-	resp, err := http.Get(latestVersionURL)
+	resp, err := http.Get(url)
 	if err != nil {
 		return none, none, errors.Wrap(err, "getting latest version info from GCS")
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
-		return none, none, errors.Wrapf(err, "http %d, error: %s", resp.StatusCode, resp.Status)
```

Comment:
`err` will be always nil since it is sure `err` is nil as [line 50](https://github.com/GoogleContainerTools/skaffold/pull/1636/files#diff-3bf14b4d33cbb50151fe5326e8cb450bL50) suggests,  and `errors.Wrapf` returns nil when err is nil.